app/src/main/res/layout/activity_main.xml

```diff
@@ -33,6 +33,19 @@
         app:layout_constraintTop_toTopOf="parent"
         tools:ignore="ContentDescription" />
 
+    <ImageButton
+        android:id="@+id/btn_theme_mode"
```

Comment:
Need to do this button with border like this: @drawable/ic_result_button". And need revers icons in buttons. When darkTheme -> show sun, when lightThem -> show moon.